packages/fether-react/src/Accounts/CreateAccount/AccountImportOptions/AccountImportOptions.js

```diff
@@ -115,14 +161,20 @@ class AccountImportOptions extends Component {
     });
   };
 
-  hasExistingAddressForImport = (addressForImport, chainId) => {
+  isCurrentChainIdTheAddressForImportChainId = chainIdInt => {
```

Comment:
as I wrote before I would remove this function altogether

Reply:
i've addressed this issue in commit c4cf7d580da8e2926d78b2092760300afe2fedfd